views/upload_form.pug

```diff
@@ -0,0 +1,9 @@
+doctype 5
```

Comment:
What exactly is the purpose of this file?

Reply:
Nothing, was trying stuff out with image upload. Removed